Extension/src/extensionActivationInformation.ts

```diff
@@ -0,0 +1,44 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
This module isn't really about activation, so it would be better named "installationInformation.ts"